src/Benchmarks/Benchmarks.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.0;net451</TargetFrameworks>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Should move this into build\dependencies.props. Because we don't ship these packages, there's no reason to build against lower framework versions elsewhere in this repo.